cmake/EnableCoverage.cmake

```diff
@@ -0,0 +1,53 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# For the GCC and Clang compiler families enable a Coverage build type.
+if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang" OR
```

Comment:
I believe the style we're following is to have a space between `if` and `(` and similarly for `else (` below as well.

Reply:
Fixed.